app/views/reasons/ConfirmChangeReasonDetailView.scala.html

```diff
@@ -21,15 +21,15 @@
         button: components.button
 )
 
-@(changeReason: SummaryList, boxNumber: Int)(implicit request: Request[_], messages: Messages)
+@(changeReason: SummaryList, pageTitle: String, heading: String)(implicit request: Request[_], messages: Messages)
 
 @layout(
-    pageTitle = messages("confirmChangeReason.pageTitle", boxNumber),
+    pageTitle = pageTitle,
     customBackLinkUrl = None,
     showBackLink = false
 ) {
 
-    @h1(messages("confirmChangeReason.heading", boxNumber))
+    @h1(heading)
 
     @govukSummaryList(changeReason)
 
```

Comment:
Could you update the button from "Continue" to "Confirm" as per the content document (VD63)

https://docs.google.com/presentation/d/1U2byv7PC33zhNS0KwewcCH5-tU4sPaQfxMyrpvwTFAw/edit#slide=id.gcc35b0bd6a_0_435

Reply:
done